e2e/features/action-bar-basic.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Action bar basic
+	Scenario: Page title is Look People Basic
+		Given I am on "/action-bar-basic"
```

Comment:
Can we do like `I am on action-bar-basic` without quotes?

Reply:
No